src/sass/messaging/partials/_messaging-count-nav.scss

```diff
@@ -30,6 +30,7 @@
   }
 
   &[disabled] {
+    &,
```

Comment:
Since `&` takes the form of the selector it's nested under, `&[disabled]` here is `.messaging-message-nav button[disabled]`, and `&` again under that is the same (`.messaging-message-nav button[disabled]`).